admin/stats_sales_report_graphs.php

```diff
@@ -55,15 +55,16 @@
 
 // check start and end Date
 $startDate = "";
-if (($_GET['startDate']) && (zen_not_null($_GET['startDate']))) {
+if (isset($_GET['startDate']) && $_GET['startDate'] && zen_not_null($_GET['startDate']) && $_GET['startDate'] >= '0001-01-01') {

```

Comment:
Can't a simple `empty()` check be used here?

Is there actually a case where startDate/endDate could ever be 0001-01-01 ? 
Or were you just applying patterns from other files?

From what I can see, the `$_GET['startDate']` (and endDate) are generated by the class response data, as links for filtering, but I'm not seeing where it ever sets `0001-01-01`. (I do see it passing timestamps.)
Am I missing something?

Reply:
Was thinking of acceptable patterns on the `$_GET` considering the information is on the URI and not necessarily always having to come from the identified code.

> Can't a simple `empty()` check be used here?
Yes more than likely. Been hesitant in some cases of compressing for that type of check thinking that perhaps `zen_not_null` might actually do some sort of a date check at some point and is/was important.